internal/dashboard/business/oceanbase/obcluster.go

```diff
@@ -127,7 +127,7 @@ func buildOBClusterResponse(ctx context.Context, obcluster *v1alpha1.OBCluster)
 		respCluster.Monitor.Image = obcluster.Spec.MonitorTemplate.Image
 		respCluster.Monitor.Resource = response.ResourceSpecRender{
 			Cpu:      obcluster.Spec.MonitorTemplate.Resource.Cpu.Value(),
-			MemoryGB: obcluster.Spec.MonitorTemplate.Resource.Memory.String(),
+			MemoryGB: fmt.Sprintf("%d GB", obcluster.Spec.MonitorTemplate.Resource.Memory.Value()>>30),
```

Comment:
return the actual value and parse it in frontend

Reply:
I don't think it's necessary to return the original value here. The response is just of type `ResourceSpecRender` for rendering. And the frontend need to change its code if we return original value here.